apps/account-ui/src/app.module.css

```diff
@@ -0,0 +1,4 @@
+
+img.logo {
```

Comment:
Let's not make this selector overly specific.
```suggestion
.logo {
```

Reply:
it's needed otherwise it doesn't override patternfly